Kitodo/rulesets/subhh.xml

```diff
@@ -0,0 +1,1207 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!--
+ *
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ *
+-->
+<ruleset xmlns="http://names.kitodo.org/ruleset/v2"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://names.kitodo.org/ruleset/v2 ../../Kitodo-DataEditor/src/main/resources/ruleset.xsd">
```

Comment:
Rulesets are normally copied to a local configuration directory, so this relative path to a subfolder of `Kitodo-DataEditor` won't normally work, I think.

Reply:
So what about this?

Reply:
I would concider it the best to address the XSD file through an URL. This could be the GitHub URL (`https://raw.githubusercontent.com/kitodo/kitodo-production/master/Kitodo-DataEditor/src/main/resources/ruleset.xsd`), but I would prefer it on the web server `goobi.org`. What do you think?

Reply:
Which webserver goobi.org? :-D I think, you are meaning kitodo.org.

Reply:
How about putting `ruleset.xsd` into the same directory (`Kitodo/rulesets/`) until it's available via some URL? Then it would (hopefully) be clearer that this ruleset has to be copied as well when the user puts some custom ruleset file into his local ruleset configuration directory? And it could be referenced from the custom ruleset files without an awkward relative path.

Reply:
^^ You know that yesterday was Rose Monday, right? :smirk:
So it will go into the `rulesets/` directory then.